rosette-scala/src/main/scala/coop/rchain/rosette/Location.scala

```diff
@@ -136,4 +136,103 @@ object Location {
 
       case LTLimbo => Ob.INVALID
     }
+
+  def printRep(loc: Location): String = {
+    val names = List("rslt",
+                     "trgt",
+                     "argvec",
+                     "env",
+                     "code",
+                     "ctxt",
+                     "self",
+                     "self-env",
+                     "rcvr",
+                     "monitor")
+
+    loc.genericType match {
+      case LTCtxtRegister(reg) =>
+        if (0 < reg && reg < Location.NumberOfCtxtRegs) {
+          names(reg)
+        } else {
+          s"unknown ctxt register 0x$reg%x"
+        }
+
+      case LTArgRegister(argReg) =>
+        s"arg[$argReg]"
+
+      case LTLexVariable(indirect, level, offset) => {
```

Comment:
I think you mixed up lines here? Should be something with `lex[%d,(%d)]`.

Reply:
Fixed.